src/foundry/common/abstract/_types.d.mts

```diff
@@ -0,0 +1,246 @@
+export interface DatabaseGetOperation {
```

Comment:
If you wanted to be more DRY you could add another interface that all the database operations extend from.

Reply:
Implemented these straight from the official docs, could look into how many of the properties are just repeats.